specs/lib/loadGltfUrisSpec.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+var fs = require('fs');
+var loadGltfUris = require('../../lib/loadGltfUris');
+var fragmentShaderPath = './specs/data/boxTexturedUnoptimized/CesiumTexturedBoxTest0FS.glsl';
+
+describe('loadGltfUris', function() {
+    var fragmentShaderData;
+
+    beforeAll(function(done) {
+        fs.readFile(fragmentShaderPath, function (err, data) {
+            if (err) {
+                throw err;
+            }
+            fragmentShaderData = data;
+            done();
+        });
+    });
+    
+    it('loads an external shader', function(done) {
```

Comment:
To be extra careful testing the async code, I think it would be useful to add another test with two `shaders` (could be the same GLSL source): one that points to a .glsl file and another with a data uri.